### PR TITLE
add issues link to tip

### PR DESCRIPTION
**Pull request by CocosGames**

### CODE REVIEW

File: docs/guide/help.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Project V 提供了下列讨论组，用于讨论不同类型的问题。
 `security@v2fly.org`: 仅用于沟通涉及安全隐患的内容。
 
 ::: tip
-由于工作繁忙，不能保证每封邮件都回复，请见谅。请尽量使用社区讨论以得到更快速的社区响应。
+由于工作繁忙，不能保证每封邮件都回复，请见谅。请尽量使用[社区讨论](https://github.com/v2fly/v2ray-core/issues)以得到更快速的响应。
```
Comment (DuckSoft) [inline docs/guide/help.md:42]:
```suggestion
由于工作繁忙，不能保证每封邮件都回复，请见谅。请尽量使用[社区讨论](https://github.com/v2fly/discussion/issues)以得到更快速的响应。
```

Comment (DuckSoft) [reply docs/guide/help.md:42]: Need confirmation from @v2fly/developers

Comment (kslr) [reply docs/guide/help.md:42]: 我觉得可以保持先有状态，等 GitHub Discussions 推出来后就不需要 discussion 了

Comment (DuckSoft) [reply docs/guide/help.md:42]: Fine. I will leave it as is.

**Pull request merged**